css/style.css

```diff
@@ -0,0 +1,2867 @@
+    background-size: 822px 271px;
+  }
+
+  .top__title {
+    margin-left: 0;
+    margin-top: 55px;
+  }
+
+  .top__subtitle {
+    margin-left: 0;
+    margin-right: 0;
+  }
+
+  .top__img {
+    width: 700px;
+    height: auto;
+    margin-right: 0;
+    margin-top: 89px;
+    margin-left: 0;
+  }
+
+  .top__wrapper {
+    display: flex;
+    flex-direction: column;
+    margin-left: 10px;
+    margin-top: 44px;
+  }
+
+  .top__wrapper-charset {
+    background: url(../img/svg/road-ipad.svg) no-repeat;
+    background-position: 413px 1px;
+    margin-top: 0;
+  }
+
+  .top__wrapper-item {
+    display: flex;
+    justify-content: center;
+    margin-top: 45px;
+    margin-bottom: 55px;
+    margin-left: 15px;
+    margin-right: 0;
+  }
+
+  .charset {
+    margin-right: 34px;
+    margin-left: 30px;
+    justify-content: center;
+  }
+
+  .charset__speed {
+    width: 65.92px;
+    height: 50.16px;
+    padding-left: 0;
+    margin-left: 98px;
+    margin-right: 4px;
+    margin-top: 4px;
+  }
+
+  .charset__road {
+    display: none;
+    width: 37.83px;
+    height: 60.72px;
+    margin-left: 20px;
+  }
+
+  .charset__first {
+    align-items: center;
+    margin-bottom: 9px;
+  }
+
+  .charset__first-big {
+    font-weight: 600;
+    font-size: 50px;
+    padding-left: 15px;
+    padding-bottom: 5px;
+  }
+
+  .charset__first-small {
+    margin-right: 62px;
+    padding-right: 0;
+    padding-left: 4px;
+    padding-bottom: 0;
+    padding-top: 7px;
+  }
+
+  .charset__second {
+    padding-left: 15px;
+    margin-bottom: 0;
+  }
+
+  .button {
+    margin: 0;
+    justify-content: center;
+  }
+
+  .button__outline {
+    color: #e51937;
+    margin-right: 3px;
+    margin-left: 18px;
+    padding-bottom: 10px;
+    padding-top: 16px;
+    padding-right: 30px;
+    padding-left: 45px;
+    background: url(../img/svg/steering-ipad.svg) no-repeat;
+    background-position: 1px 8px;
+  }
+
+  .button__outline-img {
+    display: none;
+    padding-left: 28px;
+    margin-right: 15px;
+    margin-left: 0;
+  }
+
+  .button__text {
+    width: 110px;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 20px;
+  }
+
+  .button__red {
+    background-color: #e51937;
+    color: #fff;
+    margin: 0 30px;
+    justify-content: center;
+    margin-left: 0;
+    padding-left: 50px;
+    padding-right: 47px;
+    padding-top: 15px;
+    padding-bottom: 15px;
+    margin-right: 45px;
+    margin-bottom: 0;
+  }
+
+  .middle {
+    max-width: 768px;
+    margin-top: 30px;
+  }
+
+  .middle__inner {
+    align-content: space-between;
+    background: url(../img/tesla-fon-ipad-2.png) no-repeat;
+  }
+
+  .middle__fon {
+    display: flex;
+    display: inline-block;
+  }
+
+  .middle__title {
+    margin-top: 11px;
+    margin-left: 434px;
+  }
+
+  .middle__info {
+    max-width: 768px;
+    height: 490px;
+    margin-top: 405px;
+  }
+
+  .middle__info-row {
+    display: flex;
+    flex-direction: row;
+  }
+    
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)